bot/openai_utils.py

```diff
@@ -134,16 +247,32 @@ def _generate_prompt(self, message, dialog_messages, chat_mode):
 
         return prompt
 
-    def _generate_prompt_messages(self, message, dialog_messages, chat_mode):
+    def _encode_image(self, image_buffer: BytesIO) -> bytes:
+        return base64.b64encode(image_buffer.read()).decode("utf-8")
+
+    def _generate_prompt_messages(self, message, dialog_messages, chat_mode, image_buffer: BytesIO = None):
         prompt = config.chat_modes[chat_mode]["prompt_start"]
 
         messages = [{"role": "system", "content": prompt}]
+        user_messages = {"role": "user", "content": []}
+        
         for dialog_message in dialog_messages:
-            messages.append({"role": "user", "content": dialog_message["user"]})
+            user_messages["content"].extend(dialog_message["user"])
```

Comment:
This broke compatibility with `message_handle_fn`. Created pull request with proposed fix https://github.com/father-bot/chatgpt_telegram_bot/pull/449